src/asm/section.c

```diff
@@ -339,6 +339,9 @@ void out_SetLoadSection(char const *name, uint32_t type, uint32_t org,
 	if (currentLoadSection)
 		fatalerror("`LOAD` blocks cannot be nested\n");
 
+	if (sect_HasData(type))
+		fatalerror("`LOAD` blocks cannot be placed within a ROM section\n");
```

Comment:
Is there a way to make it non-fatal without too much modification? `return` there, but then what with the closing function?

Reply:
It could be non-fatal, no `return`, and just do whatever it currently does. Or it could `return` and stay within the current ROM section (so then the `ENDL` would complain "Found `ENDL` outside of a `LOAD` block").

Reply:
I think that just continuing the function after the error would make the most sense. It would then add the LOADed data to the enclosing ROM section (not to do that would be weird and IMO worth being fatal). How about it?

Reply:
`LOAD` to `ROM0`/`ROMX` is now a non-fatal error; the closing `ENDL` is fine with that.